src/book/03-guides/03-hardware/01-working-with-robots.mdx

```diff
@@ -8,14 +8,8 @@ slug: /guides/hardware/working-with-robots
 
 ## Aligning an Ankle
 
-<<<<<<< HEAD
 An alignment of an ankle may need to be done if the foot and the leg of the robot aren't perpendicular when standing.
 An ankle alignment may also be necessary when the foot has come loose.
-=======
-An alignment of an ankle may need to be done if the foot and the leg of the robot aren't perpendicular when standing.
-An ankle alignment may also be necessary when the foot has come loose.
-
-> > > > > > > 8f25ae92838312b8020ba3e2a14bd2ceeaf324c5
```

Comment:
This is what a git merge conflict looks like. You need to choose the section that has the correct content and delete the other section (or create some mixture of the two sections). You need to also remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> 8f25ae92838312b8020ba3e2a14bd2ceeaf324c5` bits

Reply:
Are they still on my latest commit?

Reply:
Oh, nevermind. I haven't had enough coffee yet :P